composer.json

```diff
@@ -6,7 +6,15 @@
             "php": "5.6"
         }
     },
+    "require": {
+    },
     "require-dev": {
-        "owncloud/coding-standard": "^1.0"
+        "phpunit/phpunit": "^5.7",
```

Comment:
why? core's phpunit is used - isn't it?

Reply:
I pasted this from ``password_policy``
I will try without it.

Reply:
phpunit works without this :+1: